experimental/framework/frs-client/src/frsClient.ts

```diff
@@ -0,0 +1,157 @@
+    constructor(private readonly serviceConnectionConfig: RouterliciousService) {
+    }
+
+    public async createContainer(
+        serviceContainerConfig: FRSContainerConfig,
+        containerSchema: ContainerSchema,
+    ): Promise<FluidContainer> {
+        const runtimeFactory = new DOProviderContainerRuntimeFactory(
+            containerSchema,
+        );
+        const container = await this.getContainerCore(
+            serviceContainerConfig,
+            runtimeFactory,
+            true,
+        );
+
+        const rootDataObject = await this.getRootDataObject(container);
+        return new FluidContainer(container, rootDataObject);
+    }
+
+    public async getContainer(
+        serviceContainerConfig: FRSContainerConfig,
+        containerSchema: ContainerSchema,
+    ): Promise<FluidContainer> {
+        const runtimeFactory = new DOProviderContainerRuntimeFactory(
+            containerSchema,
+        );
+        const container = await this.getContainerCore(
+            serviceContainerConfig,
+            runtimeFactory,
+            false,
+        );
+
+        const rootDataObject = await this.getRootDataObject(container);
+        return new FluidContainer(container, rootDataObject);
+    }
+
+    private async getRootDataObject(
+        container: Container,
+    ): Promise<RootDataObject> {
+        const rootDataObject = (await container.request({ url: "/" })).value;
+        return rootDataObject as RootDataObject;
+    }
+
+    private async getContainerCore(
+        routerliciousContainerConfig: FRSContainerConfig,
+        containerRuntimeFactory: IRuntimeFactory,
+        createNew: boolean,
+    ): Promise<Container> {
+        const module = { fluidExport: containerRuntimeFactory };
+        const codeLoader = { load: async () => module };
+
+        const loader = new Loader({
+            urlResolver: this.serviceConnectionConfig.urlResolver,
+            documentServiceFactory: this.serviceConnectionConfig.documentServiceFactory,
+            codeLoader,
+            logger: routerliciousContainerConfig.logger,
+        });
+
+        let container: Container;
+
+        if (createNew) {
+            // We're not actually using the code proposal (our code loader always loads the same module
+            // regardless of the proposal), but the Container will only give us a NullRuntime if there's
+            // no proposal.  So we'll use a fake proposal.
+            container = await loader.createDetachedContainer({
+                package: "no-dynamic-package",
+                config: {},
+            });
+            await container.attach({ url: routerliciousContainerConfig.id });
+        } else {
+            // Request must be appropriate and parseable by resolver.
+            container = await loader.resolve({ url: routerliciousContainerConfig.id });
+            // If we didn't create the container properly, then it won't function correctly.  So we'll throw if we got a
```

Comment:
Can we instead of this comment and check below bake it directly into Loader? I believe we no longer supporting creating files on request (or rather maybe - should not, as the only supported path today is detached creation)